opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -637,12 +642,6 @@ namespace Opm {
             if (well_collection_->requireWellPotentials()) {
 
                 // calculate the well potentials
-                setWellSolutions(well_state);
```

Comment:
I guess this is neither triggered by Norne nor Model 2. But I still wonder why these computations are now not needed any more. Can someone enlighten me?

Reply:
Hi, they are still needed, while it is happening on the individual well level, so you will not see them in the `StandardWellsDense` level.  The corresponding functions are `initPrimaryVariablesEvaluation` now for a slightly better readability. Naming was pretty difficult sometimes and they are still under improvement.